src/Core/src/Handlers/ActivityIndicator/ActivityIndicatorHandler.cs

```diff
@@ -6,7 +6,7 @@
 using PlatformView = Microsoft.UI.Xaml.Controls.ProgressRing;
 #elif TIZEN
 using PlatformView = ElmSharp.ProgressBar;
-#elif NETSTANDARD || (NET6_0 && !IOS && !ANDROID && !TIZEN)
+#elif (NETSTANDARD || NET6_0) || (NET6_0 && !IOS && !ANDROID && !TIZEN)
```

Comment:
Here is a really weird one that probably should be simplified, but not doing that here.

This is incorrect because we basically ALWAYS det this condition to be true, but we don't because the more specific ones are first. The correct condition is probably:

```cs
#elif !PLATFORM
```
since we defined `PLATFORM` to only exist when there is a platform - so not `NETSTANDARD` or plain `NET6_0`

Reply:
In fact, this should just be a plain else. All the platforms are before it so I am not sure why we even have an if here...